### ISSUE TITLE
Abnormal Loss curve

### ISSUE THREAD

**Issue by cwchenwang** [cvlab-columbia/zero123#49]
Hi, I trained with 4 A100s with gradient accumulation of 2. However, the loss curve seems to be not right:

![image](https://user-images.githubusercontent.com/23579918/235353629-1bbab2b3-9d84-463d-b174-1c9fe429af6e.png)

This is the reconstruction results:
![image](https://user-images.githubusercontent.com/23579918/235353657-11079c2e-6c91-4e85-bc4d-d566e9bb92ef.png)

**Comment by ruoshiliu** [comment]
I can't really help you without any additional information... Can you give more details?

**Comment by cwchenwang** [comment]
Thanks for your reply. I just trained with the default config except setting gradient accumulation to 2. Now I have trained for about 20k iterations, and the loss is about 0.021. Just not sure if this is normal since the loss curve is fluctuating like the image I show above

**Comment by ruoshiliu** [comment]
I think loss step is supposed to pretty unstable but loss_epoch should consistently decrease with much smaller variance than loss_step.
